core/rpc/json/user/commands.go

```diff
@@ -91,6 +91,8 @@ type EstimatePriceRequest struct {
 type QueryRequest struct {
 	DBID  string `json:"dbid"`
 	Query string `json:"query"`
+
+	Challenge *types.SignedChallenge `json:"challenge"`
```

Comment:
`omitempty` in the JSON tag.  We don't want `"challenge": null` in every request that is not using authenticated RPCS.